lib/huk_load/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+if (CONFIG_HUK_LOAD)
```

Comment:
This check is not needed here since the parent cmakelist does  `add_subdirectory_ifdef(CONFIG_HUK_LOAD ...`